Lib/fontTools/subset/__init__.py

```diff
@@ -1789,6 +1789,13 @@ def subset_glyphs(self, s):
 	self.glyphCount = len(self.variations)
 	return bool(self.variations)
 
+def _remap_index_map(s, varidx_map, table_map):
+	map = {k:varidx_map[v] for k,v in table_map.mapping.items()}
+	last_idx = varidx_map[table_map.mapping[s.last_retained_glyph]]
+	for g,i in s.reverseEmptiedGlyphMap.items():
+		map[g] = last_idx if i > s.last_retained_order else 0
```

Comment:
you could add a comment here explaining why the varIdx for emptied glyphs are set to either 0/0 or to the varIdx of the last retained glyph (e.g. mention the fact they are optimized out as table is compiled).

Reply:
`map` is a built-in fuction name; most syntax highlighter can't distinguish between builtins and local variables that shadow the latter (as you see it appears in blue). It's distracting when reading at code. Conventionally one would add an underscore to the name e.g. `map_` or use a different name altogether

Reply:
added comment & renamed map to map_